packages/panzer/mini-em/example/BlockPrec/solverMueLuRefMaxwellCuda.xml

```diff
@@ -12,66 +12,41 @@
-                  <Parameter name="relaxation: backward mode" type="bool" value="false"/>
-                </ParameterList>
-                <Parameter name="smoother: post type" type="string" value="RELAXATION"/>
-                <ParameterList name="smoother: post params">
-                  <Parameter name="relaxation: type" type="string" value="MT Gauss-Seidel"/>
-                  <Parameter name="relaxation: symmetric matrix structure" type="bool" value="true"/>
-                  <Parameter name="relaxation: sweeps" type="int" value="2"/>
-                  <Parameter name="relaxation: use l1" type="bool" value="true"/>
-                  <Parameter name="relaxation: backward mode" type="bool" value="true"/>
+                <Parameter name="smoother: type" type="string" value="CHEBYSHEV"/>
+                <ParameterList name="smoother: params">
+                  <Parameter name="chebyshev: degree" type="int" value="2"/>
+                  <Parameter name="chebyshev: ratio eigenvalue" type="double" value="5.4"/>
                 </ParameterList>
 
                 <ParameterList name="refmaxwell: 11list">
                   <Parameter name="use kokkos refactor" type="bool" value="true"/>
                   <Parameter name="tentative: calculate qr" type="bool" value="false"/>
                   <Parameter name="rap: triple product" type="bool" value="false"/>
 
-                  <Parameter name="smoother: pre type" type="string" value="RELAXATION"/>
-                  <ParameterList name="smoother: pre params">
-                    <Parameter name="relaxation: type" type="string" value="MT Gauss-Seidel"/>
-                    <Parameter name="relaxation: symmetric matrix structure" type="bool" value="true"/>
-                    <Parameter name="relaxation: sweeps" type="int" value="2"/>
-                    <Parameter name="relaxation: use l1" type="bool" value="true"/>
-                    <Parameter name="relaxation: backward mode" type="bool" value="false"/>
-                  </ParameterList>
-                  <Parameter name="smoother: post type" type="string" value="RELAXATION"/>
-                  <ParameterList name="smoother: post params">
-                    <Parameter name="relaxation: type" type="string" value="MT Gauss-Seidel"/>
-                    <Parameter name="relaxation: symmetric matrix structure" type="bool" value="true"/>
-                    <Parameter name="relaxation: sweeps" type="int" value="2"/>
-                    <Parameter name="relaxation: use l1" type="bool" value="true"/>
-                    <Parameter name="relaxation: backward mode" type="bool" value="true"/>
+                  <Parameter name="smoother: type" type="string" value="CHEBYSHEV"/>
+                  <ParameterList name="smoother: params">
+                    <Parameter name="chebyshev: degree" type="int" value="2"/>
+                    <Parameter name="chebyshev: ratio eigenvalue" type="double" value="5.4"/>
```

Comment:
5.4?

Reply:
Found through a very rigorous testing procedure ;-)
I quickly tested this on white, and this seemed to be faster than MT GS (despite 15% more iterations).
I should probably think at some point a bit more about analytical ways of getting to a good value.